File: sdk/dart/lib/model/order.dart
part of swagger.api;


@Entity()
class Order {
  
  @Property(name: 'ID')
  String ID = null;
  

  @Property(name: 'Type')
  String type = null;
  

  @Property(name: 'FromUserID')
  String fromUserID = null;
  

  @Property(name: 'BillingAddressID')
  String billingAddressID = null;
  

  @Property(name: 'ShippingAddressID')
  String shippingAddressID = null;
  

  @Property(name: 'Comments')
  String comments = null;
  

  @Property(name: 'ShippingCost')
  Number shippingCost = null;
  

  @Property(name: 'TaxCost')
  Number taxCost = null;
  

  @Property(name: 'xp')
  Object xp = null;
  
  Order();

  @override
  String toString()  {
    return 'Order[ID=$ID, type=$type, fromUserID=$fromUserID, billingAddressID=$billingAddressID, shippingAddressID=$shippingAddressID, comments=$comments, shippingCost=$shippingCost, taxCost=$taxCost, xp=$xp, ]';
  }

}

File: sdk/slim/lib/Models/Order.php
<?php
/*
 * Order
 */
namespace \Models;

/*
 * Order
 */
class Order {
    /* @var string $iD  */
    private $iD;
/* @var string $type  */
    private $type;
/* @var string $fromUserID  */
    private $fromUserID;
/* @var string $billingAddressID  */
    private $billingAddressID;
/* @var string $shippingAddressID  */
    private $shippingAddressID;
/* @var string $comments  */
    private $comments;
/* @var Number $shippingCost  */
    private $shippingCost;
/* @var Number $taxCost  */
    private $taxCost;
/* @var object $xp  */
    private $xp;
}
